Gems/ROS2/Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -67,12 +67,20 @@ namespace ROS2
             AZStd::lock_guard<AZStd::mutex> lck(m_statusLock);
             m_status.clear();
         }
+
+         // Begin an undo batch for prefab creation process
```

Comment:
This variable needs to be initialized to `nullptr` as an EBus call might not be listening and therefore the result variable isn't updated.